pkg/constants/constants.go

```diff
@@ -116,4 +103,24 @@ const (
 var (
 	// JobCompletionIndexFieldPath is the field path for the Job completion index annotation.
 	JobCompletionIndexFieldPath string = fmt.Sprintf("metadata.annotations['%s']", batchv1.JobCompletionIndexAnnotation)
+
+	// Distributed envs for torchrun.
+	// Ref: https://github.com/pytorch/pytorch/blob/3a0d0885171376ed610c8175a19ba40411fc6f3f/torch/distributed/argparse_util.py#L45
+	// TorchEnvNumNodes is the env name for the number of training nodes.
+	TorchEnvNumNodes string = fmt.Sprintf("%sNNODS", TorchEnvNamePrefix)
```

Comment:
Since we are no longer using the `TorchEnvNamePrefix` variable, it might be easier to just set the full variable right now, so it is easier to read.
```suggestion
	// Distributed envs for torchrun.
	// Ref: https://github.com/pytorch/pytorch/blob/3a0d0885171376ed610c8175a19ba40411fc6f3f/torch/distributed/argparse_util.py#L45
	// TorchEnvNumNodes is the env name for the number of training nodes.
	TorchEnvNumNodes string = "PET_NNODES"
```